docs/docs-beta/docs/guides/pipelines.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Create a pipeline"
+description: "Learn how to create data pipelines using Dagster's asset-based approach"
+---
+
+In Dagster, data pipelines are created using an asset-based approach. This overview will introduce you to the key concepts and steps involved in building a Dagster pipeline.
+
+## What you'll learn
+
+- The basics of Dagster's asset-based pipelines
+- How to define and connect data assets
+- Ways to enhance your pipeline with metadata and partitioning
+- How to integrate external data sources
+
+<details>
+  <summary>Prerequisites</summary>
+
+Before continuing, you should be familiar with:
+
+- [Installing Dagster](/tutorial/installation)
+- [The Quickstart Tutorial](/tutorial/quick-start)
+
+</details>
+
+Dagster uses software-defined assets as the building blocks of data pipelines. An asset represents a data object, such as a table, file, or machine learning model, that your pipeline produces or updates.
+
+As you define multiple assets and their dependencies, Dagster automatically creates an asset graph. This graph represents the structure and flow of your data pipeline. These nodes are not the individual operations that create the assets, but rather the assets themselves.
+
+```mermaid
+flowchart LR
+    A[Raw Sales Data] --> B[Cleaned Sales Data]
+    A -.-> C([Data Quality Checks])
```

Comment:
Visually this struck me as a little odd. It took a second to realize that this wasn't actually an asset, even though it's a different shape and has a dashed connecting line vs solid.

- Should we differentiate this more?
- It's not mentioned in the steps below, which sorta makes it stand out to me - should it be included?